cpp/test/comm/idl.cpp

```diff
@@ -0,0 +1,3 @@
+#include <unit_test_comm_apply.cpp>
```

Comment:
I was matching most of cpp/test/core/idl.cpp. Want me to change both files in this commit to dquotes?